default.nix

```diff
@@ -35,14 +35,12 @@ in makerScriptPackage {
     dss-deploy
   ];
 
-  # Patch scripts by removing `cd` commands and adding DAPP_OUT env so that
-  # `dapp create` can find the right binaries even though no submodules have
-  # been downloaded inside the Nix build environment.
+  # Patch scripts by removing `cd` commands and `./bin/` from `dss-deploy`
```

Comment:
Is this about `dss-deploy`, or TDDS? If TDDS, aren't we removing the `cd` invocations in this PR?

If `dss-deploy`, I suppose we want to port this PR over there too?

Reply:
Sorry, that comment is a bit vague, I agree.

This PR isn't removing the `cd` and `./bin/` statements that are used to enter and execute `dss-deploy` in `tdds`. And there is another issue to tackle this #35 

Yes this PR should be copied over to `dss-deploy` (also mentioned in #35)